### PR TITLE
[onert] Add quant8 support for slice cker

### PR DESCRIPTION
**Pull request by dev-hue**
Add quant8 support for slice cker

ONE-DCO-1.0-Signed-off-by: Hyo Jong Kim hue.kim@samsung.com

### CODE REVIEW

**Comment by dev-hue** [comment]
@nnfw-bot test all

**Comment by dev-hue** [comment]
@nnfw-bot test all

File: tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu [hunk 1]
```diff
@@ -125,7 +125,6 @@ GeneratedTests.select_v1_2_one_dim_quant8
 GeneratedTests.select_v1_2_two_dim_quant8
 GeneratedTests.slice_5
 GeneratedTests.slice_6
-GeneratedTests.slice_7
```
Comment (btheosam) [inline tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:128]: Why only slice_7 is enabled?

File: tests/nnapi/nnapi_gtest.skip.armv7l-linux.cpu [hunk 2]
```diff
@@ -125,7 +125,6 @@ GeneratedTests.select_v1_2_one_dim_quant8
 GeneratedTests.select_v1_2_two_dim_quant8
 GeneratedTests.slice_5
 GeneratedTests.slice_6
-GeneratedTests.slice_7
```
Comment (btheosam) [inline tests/nnapi/nnapi_gtest.skip.armv7l-linux.cpu:128]: Same here too.

File: tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu (hunk 1 above)
Comment (hasw7569) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:128]: 5,6,8 seems to int32 tests.
but we don't have int kernel yet.
so I think it is fine to disable 5,6,8.

Comment (dev-hue) [reply tests/nnapi/nnapi_gtest.skip.aarch64-linux.cpu:128]: @btheosam 
Refer to `slice.mod.py`. There is eight of testcase.
The 7th is about quant8.

**Pull request merged**